### PR TITLE
[FIF-8] Initial TeamCity Config Setup

### PR DESCRIPTION
**Pull request by stephenfuqua**

### CODE REVIEW

File: .teamcity/pom.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0"?>
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <name>EdFi_AnalyticsTeam_FixItFriday Config DSL Script</name>
```
Comment (kurtjohnson-HS) [inline .teamcity/pom.xml:4]: Question on underscores - if this is for readabilityh, we could consider editing these underscores to spaces.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.teamcity/target/
```
Comment (kurtjohnson-HS) [inline .gitignore:1]: We will want to merge this line with the next PR that adds more gitignore entries.

File: .teamcity/pom.xml (hunk 1 above)
Comment (stephenfuqua) [reply .teamcity/pom.xml:4]: This part was created automatically by TeamCity and only TeamCity ever "sees" it.

File: .gitignore (hunk 2 above)
Comment (stephenfuqua) [reply .gitignore:1]: I see that I need to rebase now.

Comment (stephenfuqua) [reply .gitignore:28]: Done

File: .teamcity/pom.xml (hunk 1 above)
Comment (stephenfuqua) [reply .teamcity/pom.xml:4]: In fact the underscores are actually appropriate, as the project ID in TeamCity cannot have spaces and is thus "EdFi_Analytics_FixItFriday".

**Pull request merged**